### PR TITLE
Monitoring: add Grafana

### PR DESCRIPTION
**Pull request by guidoiaquinti**
## Description
Add a Grafana stack as opt-in dependency. We've also included a sample dashboard we can use to monitor Redis

Screenshot 2022-02-11 at 14 34 34

## Type of change
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)

## How has this been tested?
Tested locally + CI

## Checklist
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: charts/posthog/tests/grafana-dashboards.yaml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+      grafana.enabled: false
+    asserts:
+      - hasDocuments:
+          count: 0
+
+  - it: the manifest should match the snapshot when using default values
+    set:
+      grafana.enabled: true
+    asserts:
+      - hasDocuments:
```
Comment (hazzadous) [inline charts/posthog/tests/grafana-dashboards.yaml:17]: The assert seems to be duplicated across the tests but could just check once? Also, is it redundant the other asserts are successful

File: charts/posthog/values.yaml [hunk 2]
```diff
@@ -643,6 +643,42 @@ serviceAccount:
...
+###
+###
+### ---- Grafana ----
+###
+###
+grafana:
+  # -- Whether to install Grafana or not.
+  enabled: false
+
+  # -- Sidecar configuration to automagically pull the dashboards from the `charts/posthog/grafana-dashboard` folder. See [official docs](http://docs.grafana.org/administration/provisioning/#dashboards) for more info.
```
Comment (hazzadous) [inline charts/posthog/values.yaml:656]: While these doc links do include useful information, it seems like https://github.com/grafana/helm-charts/blob/main/charts/grafana/README.md would be more useful.

File: ci/kubetest/test_grafana.py [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+def test_helm_install(setup, kube):
+    pass
+
+
+def test_posthog_healthy(kube):
+    is_posthog_healthy(kube)
+
+
+def test_grafana(kube):
```
[the remark this answers is not in the document]
Comment (guidoiaquinti) [reply ci/kubetest/test_grafana.py:28]: I would like but https://kubetest.readthedocs.io/en/latest/reference.html#kubetest.objects.Service.proxy_http_get doesn't seems to work

[the remark this answers is not in the document]
Comment (guidoiaquinti) [reply ci/kubetest/test_grafana.py:28]: Auth is the default (admin + autogenerated password)

**Review by hazzadous — APPROVED** [review]
Looks cool. Fine to get in, question around auth, not immediately obvious how this works, maybe a little comment could clarify.

**Pull request merged**